src/clients/client.ts

```diff
@@ -20,6 +20,7 @@ export interface WalletOptoions {
 
 export class PersistenceClient {
   public wallet: DirectSecp256k1HdWallet;
+  public signer
```

Comment:
Yes, this is how it should be.. 
But I am confused now between the 3 fields, wallet, signer and mnemonic..
Can it all be just under one field named signer? 

meaning - signer can either be wallet (derived with mnemonic, or offline or ledger even) 
I would suggest that we keep  as is in cosmjs, but a field under persistenceClient class